src/components/AbstractSideNav/AbstractSideNav.tsx

```diff
@@ -14,41 +17,121 @@ export interface IAbstractSideNavProps extends HTMLAttributes<HTMLDivElement> {
+      current ? 'bg-gray-100 text-gray-900' : 'text-gray-600 hover:bg-gray-50 hover:text-gray-900',
+      'group flex items-center px-3 py-2 text-sm font-medium rounded-md',
+      disabled && 'opacity-50 pointer-events-none',
+    );
+    const iconStyle = clsx(
+      current ? 'text-gray-500' : 'text-gray-400 group-hover:text-gray-500',
+      'flex-shrink-0 mr-3 w-6 h-6',
+    );
+    const countStyle = clsx(
+      current ? 'bg-white' : 'bg-gray-100 group-hover:bg-gray-200',
+      'inline-block ml-3 px-3 py-0.5 text-xs rounded-full',
+    );
+    return (
+      <Link key={item.name} href={href}>
+        <a className={linkStyle} aria-current={current ? 'page' : undefined}>
+          <Icon className={iconStyle} aria-hidden="true" />
+          <span className="flex-1 truncate">{item.name}</span>
+          {showCount ? <span className={countStyle}>{count}</span> : null}
+        </a>
+      </Link>
+    );
+  });
 
-        const linkStyle = clsx(
-          current ? 'bg-gray-100 text-gray-900' : 'text-gray-600 hover:bg-gray-50 hover:text-gray-900',
-          'group flex items-center px-3 py-2 text-sm font-medium rounded-md',
-          disabled && 'opacity-50 pointer-events-none',
-        );
-        const iconStyle = clsx(
-          current ? 'text-gray-500' : 'text-gray-400 group-hover:text-gray-500',
-          'flex-shrink-0 mr-3 w-6 h-6',
-        );
-        const countStyle = clsx(
-          current ? 'bg-white' : 'bg-gray-100 group-hover:bg-gray-200',
-          'inline-block ml-3 px-3 py-0.5 text-xs rounded-full',
-        );
-        return (
-          <Link key={item.name} href={href}>
-            <a className={linkStyle} aria-current={current ? 'page' : undefined}>
-              <Icon className={iconStyle} aria-hidden="true" />
-              <span className="flex-1 truncate">{item.name}</span>
-              {showCount ? <span className={countStyle}>{count}</span> : null}
-            </a>
-          </Link>
-        );
-      })}
-    </nav>
+  const getTopMenu = () => {
+    const items = navigation.map((item, index) => {
+      return {
+        id: item.name,
+        label: itemElements[index],
+        domId: `absNav-${item.name}`,
+      };
+    });
+    const currentItem = navigation.find((item) => item.href === subPage);
+    const Icon = currentItem.icon || DocumentIcon;
+    const count = getCount(currentItem.href, doc);
+    const showCount = count > 0 && currentItem.href !== Routes.SIMILAR;
+    const label = (
+      <div className="text-gray-900' : 'text-gray-600 group flex items-center mt-5 px-3 py-2 text-left hover:text-gray-900 text-sm font-medium bg-gray-100 hover:bg-gray-50 rounded-md">
+        <Icon className="flex-shrink-0 mr-3 w-6 h-6 text-gray-500" aria-hidden="true" />
+        <span className="flex-1 truncate">{currentItem.name}</span>
+        {showCount ? (
+          <span className={'bg-white inline-block ml-3 px-3 py-0.5 text-xs rounded-full'}>{count}</span>
```

Comment:
Probably identical behavior, but should switch this to be a string here for `className`

Reply:
Changed to string